javapoet/src/test/java/com/palantir/javapoet/JavaFileTest.java

```diff
@@ -387,6 +388,258 @@ class Taco {
                         """);
     }
 
+    @Test
+    public void recordNoField() {
+        String source = JavaFile.builder(
+                        "com.palantir.tacos", TypeSpec.recordBuilder("Taco").build())
+                .skipJavaLangImports(true)
+                .build()
+                .toString();
+        assertThat(source)
+                .isEqualTo(
+                        """
+                               package com.palantir.tacos;
+
+                               record Taco() {
+                               }
+                               """);
```

Comment:
Fix the indentation of these text blocks

```suggestion
                        """
                        package com.palantir.tacos;

                        record Taco() {
                        }
                        """);
```